landing-page/content/common/statistics-format-spec.md

```diff
@@ -0,0 +1,111 @@
+---
+url: statistics-format
+toc: false
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Plain Format for Iceberg Statistics
```

Comment:
How about "Index and statistics file format"?

My rationale is that we should be clear that this is a file format that can hold both indexes and statistics. And I don't think we should necessarily make it specific to Iceberg, although that's where the spec lives. This could be used by other projects as well.